tests/test_estimate_eyes.py

```diff
@@ -111,22 +113,49 @@ def test_estimate_batch(self):
                     self.detector.detectOne(img, detect68Landmarks=landMarks == "L68")
                     for img in (OPEN_EYES_IMAGE, MIXED_EYES_IMAGE, CLOSED_EYES_IMAGE)
                 ]
-                warps = [self.warper.warp(faceDetection) for faceDetection in faceDetections]
-                landMarksTransformations = [
-                    self.warper.makeWarpTransformationWithLandmarks(faceDetection, landMarks)
+                warpWithLandmarksList = [
+                    WarpWithLandmarks(
+                        self.warper.warp(faceDetection),
+                        self.warper.makeWarpTransformationWithLandmarks(faceDetection, landMarks),
+                    )
                     for faceDetection in faceDetections
                 ]
-                anglesList = self.eyeEstimator.estimateBatch(landMarksTransformations, warps)
+                anglesList = self.eyeEstimator.estimateBatch(warpWithLandmarksList)
                 assert isinstance(anglesList, list)
-                assert len(anglesList) == len(warps)
+                assert len(anglesList) == len(faceDetections)
                 for angles in anglesList:
                     assert isinstance(angles, EyesEstimation)
                     self.assert_eyes_reply(angles.asDict())
 
+    def test_estimate_batch_mixed_landmarks(self):
+        """
+        Test eye estimator with two faces - 1st with 5 landmarks, 2nd with 68 landmarks
```

Comment:
[write small numbers as words](https://www.scribendi.com/academy/articles/when_to_spell_out_numbers_in_writing.en.html#:~:text=A%20simple%20rule%20for%20using,ten\)%20are%20written%20as%20numerals.)
```suggestion
        Test eye estimator with two faces: first with landmarks5, second with landmarks68
```
(5 and 68 are parts of the variable name)

Reply:
updated